crates/store/re_query/src/cache.rs

```diff
@@ -337,7 +337,7 @@ impl ChunkStoreSubscriber for QueryCache {
                         let key = QueryCacheKey::new(
                             chunk.entity_path().clone(),
                             timeline,
-                            component_desc.component_name,
+                            component_desc,
```

Comment:
Very much nitpicking
```suggestion
                            component_descr,
```